### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by omphalos**
Hello,

I haven't had the chance to track down the problem in Firefox yet, but these other fixes are in here:

Add rounded corners in body > div. 
Handle layout in a narrow screen. 
Disable buttons during auto-step.  
Fix README link.  
Fix set up instructions.
Add compiled js to .gitignore.

### CODE REVIEW

File: browser/setup.coffee [hunk 1]
```diff
@@ -1,2 +1,2 @@
 $(document).ready ->
-  CodeMirror.fromTextArea(document.getElementById('code'), mode: 'javascript')
+  CodeMirror.fromTextArea(document.getElementById('code'), mode: 'javascript', readOnly: true)
```
Comment (int3) [inline browser/setup.coffee]: Unless I'm misunderstanding this line, it seems like users won't be able to enter any of their own code at all. Can we set `readOnly` to toggle on and off in tandem with the 'auto step' mode?

**Comment by omphalos** [comment]
You are correct.  Unfortunately, CodeMirror doesn't support easy toggling of read-only mode but I can work on coming up with a better solution that what I have implemented here.

Also, I was wondering if you might be okay if I removed the dependency on browserify and wrapped the code with a universal module definition pattern (sort of like how esprima does it)?

**Comment by int3** [comment]
> You are correct. Unfortunately, CodeMirror doesn't support easy toggling of read-only mode but I can work on coming up with a better solution that what I have implemented here.

Seems possible, though I haven't tried it myself: https://groups.google.com/forum/?fromgroups=#!topic/codemirror/etMzO1GUZIU

I'd prefer sticking to browserify, just because I'd like to evaluate it for future projects. Also, AMD is kind of ugly if it's used without Iced, because of the extra indentation; and since I'm not too pleased with the Iced debugging experience, I don't think I'll be using either of these technologies outside of metajs. (It has been an interesting experience, though! Thanks for pulling in Iced, I wouldn't have tried it out otherwise.)

We should upgrade to the recently-released browserify v2 if possible. It seems like there would be significant space savings in the bundled code.

**Comment by omphalos** [comment]
I had found an older link saying it wasn't possible (https://groups.google.com/forum/?fromgroups=#!topic/codemirror/xJEngg1M7ck) but that link you provided seems to work very well.

**Comment by omphalos** [comment]
Hello!  Yes, the vendor file was a mistake -- sorry about that!  Regarding setup.coffee, I misunderstood what was going on there.  If you want CodeMirror to load fast as possible, then I believe just an inline one-liner is fastest, but I'm still happy just to revert my changes there and return setup.coffee in its original form if you want 100% coffee script from the non-vendor code.

**Comment by int3** [comment]
Sorry, I didn't realize that you made it an inline script. That looks good, thanks!

**Comment by int3** [comment]
I took out the Map stuff because I realized that ES6's map also does provide an items(), it's just one that we can't really use... if and when ES6 does come out we can go and fix it. Till then simpler is better.

Thanks for polishing it -- it's much nicer now without the glitching! I've already pushed the updated version to gh-pages, and I'll post the links to HN/Reddit on Monday. It's been great working with you -- cheers :)

**Comment by omphalos** [comment]
Yeah, me too, thanks!

**Pull request merged**